LayoutTests/fast/writing-mode/vorg-glyph-zero-crash.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<style>
+.test {
+    -webkit-writing-mode:vertical-rl;
+    font-family:'Hiragino Kaku Gothic ProN';
+}
+</style>
+<p>Make sure assertion does not fail for glpyh ID 0 of a font that has a "VORG" table in vertical flow.
+<div class="test">&#xFFFF;</div>
+<script>
+if (window.testRunner)
+    testRunner.dumpAsText();
+document.body.offsetTop;
+</script>
```

Comment:
This test already exists in LayoutTests/imported/blink/fast/writing-mode/vorg-glyph-zero-crash.html so this might suggest that the change is unnecessary. Otherwise we would probably have noticed some assertion failures.

Reply:
Cool! Will close this PR then..